float64 min_radius
float64 max_radius
float64 heading
float64 fov